jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java

```diff
@@ -136,7 +136,7 @@ public void invoke(long window, double xpos, double ypos) {
         glfwSetScrollCallback(context.getWindowHandle(), scrollCallback = new GLFWScrollCallback() {
             @Override
             public void invoke(final long window, final double xOffset, final double yOffset) {
-                onWheelScroll(window, xOffset, yOffset);
+                onWheelScroll(window, xOffset, yOffset * 120);
```

Comment:
I see it being used for Linux and Mac, but I cannot find it used on Windows:
https://github.com/LWJGL/lwjgl/blob/master/src/java/org/lwjgl/opengl/LinuxMouse.java#L48
https://github.com/LWJGL/lwjgl/blob/master/src/java/org/lwjgl/opengl/MacOSXNativeMouse.java#L53